README.md

```diff
@@ -1 +1,288 @@
+  margin: auto 25px;
+  padding: 15px 0 0 15px;
+  border: 3px solid #f5f6f7;
+  text-align: center;
+  width: 90%;
+  max-width: 500px;
+  background-color: #eb0d0d;
+}
+
+fieldset {
+  border: 0 none;
+  height: 100%;
+  padding: 25px;
+  margin: 10px 20px;
+}
+
+label {
+  display: inline-block;
+  font-size: 1.5rem;
+  margin-bottom: 10px;
+  font-weight: bold;
+}
+
+input {
+  display: block;
+  font-size: 2.5rem;
+  width: 100%;
+  height: 60px;
+  padding: 6px 12px;
+  margin: 10px 0;
+  line-height: 1.4;
+  color: white;
+  background-color: #393954;
+  border: 1px solid #f5f6f7;
+}
+
+button {
+  cursor: pointer;
+  margin-top: 15px;
+  text-decoration: none;
+  background-image: linear-gradient(#338e0c, #0e7508);
+  border: 3px solid #feac32;
+  padding: 10px 16px;
+  font-size: 23px;
+  width: 100%;
+}
+
+input:focus-visible,
+button:focus-visible {
+  outline: 3px solid #198eee;
+}
+
+.output {
+  color: white;
+  background-color: #f40f0f;
+  border: 3px solid #f5f6f7;
+  font-size: 2.5rem;
+  width: 90%;
+  max-width: 500px;
+  min-height: 55px;
+  margin-top: 25px;
+  padding: 15px;
+  overflow-wrap: break-word;
+  text-align: center;
+}
+
+.alert {
+  font-size: 2rem;
+  background-color: #ffadad;
+  border: 3px solid #850000;
+  color: #850000;
+}
+
+.hidden {
+  display: none;
+}
+```
+
+```JS
+
+const form = document.getElementById('form');
+const convertButton = document.getElementById('convert-btn');
+const output = document.getElementById('output');
+
+const convertToRoman = (num) => {
+  const ref = [
+    ['M', 1000],
+    ['CM', 900],
+    ['D', 500],
+    ['CD', 400],
+    ['C', 100],
+    ['XC', 90],
+    ['L', 50],
+    ['XL', 40],
+    ['X', 10],
+    ['IX', 9],
+    ['V', 5],
+    ['IV', 4],
+    ['I', 1],
+  ];
+  const res = [];
+
+  ref.forEach((arr) => {
+    while (num >= arr[1]) {
+      res.push(arr[0]);
+      num -= arr[1];
+    }
+  });
+
+  return res.join('');
+};
+
+const isValid = (str, int) => {
+  let errText = '';
+
+  if (!str || str.match(/[e.]/g)) {
+    errText = 'Please enter a valid number.';
+  } else if (int < 1) {
+    errText = 'Please enter a number greater than or equal to 1.';
+  } else if (int > 3999) {
+    errText = 'Please enter a number less than or equal to 3999.';
+  } else {
+    // No errors detected
+    return true;
+  }
+
+  // Handle error text and output styling
+  output.innerText = errText;
+  output.classList.add('alert');
+
+  return false;
+};
+
+const clearOutput = () => {
+  output.innerText = '';
+  output.classList.remove('alert');
+};
+
+const updateUI = () => {
+  const numStr = document.getElementById('number').value;
+  const int = parseInt(numStr, 10);
+  output.classList.remove('hidden');
+  clearOutput();
+  if (isValid(numStr, int)) {
+    output.innerText = convertToRoman(int);
+  }
+};
+
+form.addEventListener('submit', (e) => {
+  e.preventDefault();
+  updateUI();
+});
+
+convertButton.addEventListener('click', () => {
+  updateUI();
+});
+```
```

Comment:
- Remove all of this as it is so long.

Reply:
@ahzamir thank you dear Sir for your amazing comment .I have remove my code from readme.